tests/acceptance/pretty_print.py

```diff
@@ -1,5 +1,5 @@
 import json
 
 
-def pretty_json(label, data):
+def print_json(label, data):
```

Comment:
```suggestion
def json(label, data):
```

Reply:
Sorry, I set this PR to auto-merge on approve.

But anyway, the suggestion wouldn't work because the `pretty_print` module already imports the Python module `json`, so we can't have a function called `json` too.

Reply:
I think you can import as another name, and exposing a nice API is better then technical issue in your packgae
but it's water under the bridge